### PR TITLE
stbt lint: Miscellaneous fixes to stbt-missing-image and stbt-wait-until-callable checkers

### PR DESCRIPTION
**Pull request by drothlis**

### CODE REVIEW

**Comment by drothlis** [comment]
@wmanley Please review & merge if you're happy with it.

File: _stbt/pylint_plugin.py [hunk 1]
```diff
@@ -23,11 +23,10 @@
 from pylint.interfaces import IAstroidChecker
 
 try:
-    from astroid.node_classes import Attribute, Call, Expr, Keyword
+    from astroid.node_classes import Call, Expr, Keyword
     from astroid.scoped_nodes import ClassDef, FunctionDef
 except ImportError:
-    from astroid.node_classes import (
-        Getattr as Attribute, CallFunc as Call, Discard as Expr, Keyword)
+    from astroid.node_classes import CallFunc as Call, Discard as Expr, Keyword
```
Comment (wmanley) [inline _stbt/pylint_plugin.py]: These are pylint compatibility workarounds presumably.

Comment (wmanley) [inline _stbt/pylint_plugin.py]: Ignore me, this isn't new. I'm getting confused.

File: _stbt/pylint_plugin.py [hunk 2]
```diff
@@ -105,6 +99,22 @@ def visit_callfunc(self, node):
...
+        # returns True to everything (including `callable()`).
+        if inferred != YES:
+            failed_to_infer = False
+            if inferred.callable():
+                return True
+    if failed_to_infer:
+        if (isinstance(node, Call) and
+                _is_function_named(node.func, "functools.partial")):
+            return True
+    return False
```
Comment (wmanley) [inline _stbt/pylint_plugin.py]: I guess the proper fix would be to add an understanding of `functools.partial` to [brain_functools.py](https://github.com/PyCQA/astroid/blob/master/astroid/brain/brain_functools.py), but we'd still need to upgrade pylint after that.

Comment (wmanley) [inline _stbt/pylint_plugin.py]: See https://github.com/PyCQA/astroid/issues/125

**Pull request merged**